pkg/expr/sql/db_test.go

```diff
@@ -205,6 +206,80 @@ func TestErrorsFromGoMySQLServerAreFlagged(t *testing.T) {
 	require.Contains(t, err.Error(), "error in go-mysql-server")
 }
 
+func TestFrameToSQLAndBack_JSONRoundtrip(t *testing.T) {
+	expectedFrame := &data.Frame{
+		RefID: "json_test",
+		Name:  "json_test",
+		Fields: []*data.Field{
+			data.NewField("id", nil, []*int64{p(int64(1)), p(int64(2))}),
+			data.NewField("payload", nil, []*json.RawMessage{
+				p(json.RawMessage(`{"foo":1}`)),
+				p(json.RawMessage(`{"bar":"baz"}`)),
+			}),
+		},
+	}
+
+	sqlDB := &DB{}
```

Comment:
In previous tests this line would say:
```suggestion
	db := DB{}
```
I vote for consistency here - would like to see this test and the next one match our previous tests, as much as is possible.